src/htmx.js

```diff
@@ -760,7 +760,7 @@ return (function () {
          * @returns
          */
         function oobSwap(oobValue, oobElement, settleInfo) {
-            var selector = "#" + oobElement.id;
+            var selector = "#" + CSS.escape(oobElement.id);
```

Comment:
`escape()` is not supported by IE11.

https://caniuse.com/mdn-api_css_escape_static

Reply:
Well that settles that then. I'll close the PR. Though could you please point to the docs regarding browser support which HTMX guarantees so I won't make the same mistake next time, thanks!

Reply:
Couldn't it be an option to write our own escaping function here? To avoid relying on a IE11 incompatible API while still supporting special ids like those?